src/main/java/com/epam/ta/reportportal/auth/permissions/Permissions.java

```diff
@@ -33,20 +33,22 @@ private Permissions() {
 
 	public static final String ALLOWED_TO_REPORT = "hasPermission(#projectName.toLowerCase(), 'reporterPermission')";
 
-	public static final String PROJECT_MANAGER = "hasPermission(#projectName.toLowerCase(), 'projectManagerPermission')";
+	public static final String ASSIGNED_TO_PROJECT = "hasPermission(#projectName.toLowerCase(), 'isAssignedToProject')";
+
+	public static final String PROJECT_MANAGER =
+			"hasPermission(#projectName.toLowerCase(), 'projectManagerPermission')" + " && " + ASSIGNED_TO_PROJECT;
```

Comment:
maybe the condition that project manager is assigned to project should be checked inside 'projectManagerPermission' evaluator?

Reply:
my bad, that condition is checked in the 'BaseProjectPermission'